packages/api/lib/granule-remove-from-cmr.js

```diff
@@ -25,6 +26,7 @@ const { constructCollectionId } = require('@cumulus/message/Collections');
  * @private
  */
 const _removeGranuleFromCmr = async (granule, collectionId) => {
+  let metadata;
   log.info(`granules.removeGranuleFromCmrByGranule granule_id: ${granule.granule_id}, colletion_id: ${collectionId}`);
   if (!granule.published || !granule.cmr_link) {
     log.warn(`Granule ${granule.granule_id} in Collection ${collectionId} is not published to CMR, so cannot be removed from CMR`);
```

Comment:
Yeah, I think in this case we want to keep that `return` so that we exit and return from the function if the granule field `published` is `false` or `cmr_link` doesn't exist.